build-indices_wt/modify_gtf.py

```diff
@@ -0,0 +1,131 @@
+import argparse
```

Comment:
@kevinpalis This is not the same version of the script that's on the WARP develop branch. The WARP develop branch has the correct version. https://github.com/broadinstitute/warp/blob/develop/dockers/skylab/build-indices/modify_gtf.py